sdk/iot/Azure.Iot.Hub.Service/src/Authentication/SharedAccessSignatureConstants.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Iot.Hub.Service.Authentication
+{
+    internal static class SharedAccessSignatureConstants
+    {
+        public const int MaxKeyNameLength = 256;
+        public const int MaxKeyLength = 256;
+        public const string SharedAccessSignature = "SharedAccessSignature";
+        public const string AudienceFieldName = "sr";
```

Comment:
Can we rename this one to something better like username?

Reply:
If you look at the [shared access signature builder class](https://github.com/Azure/azure-sdk-for-net/blob/feature/abmisr/devicesE2e/sdk/iot/Azure.Iot.Hub.Service/src/Authentication/SharedAccessSignatureBuilder.cs#L58), you'll see that this is the "audience" for the sas token -> iot hub hostname